learn/app-development/custom-widgets/create-prefab-with-partials.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Creating Prefabs with Partials"
+id: ""
+sidebar_label: "Create Prefab with Partials"
+---
+
+[Prefabs](/learn/app-development/custom-widgets/prefabs-overview) are reusable application parts that can be easily integrated into your applications. To learn how Prefabs can be created in WaveMaker, see [Creating Prefabs](/learn/app-development/custom-widgets/creating-prefabs).
+
+<!--truncate-->
+
+## Need for Partials within Prefabs
+
+Until now, we did not allow users to create [partials](/learn/app-development/ui-design/page-concepts/partial-pages) inside Prefabs. The only page you have access to is the Main Page where all code exists. Due to this, the following problems can be seen:
```

Comment:
i think instead of -- "until now" --- but now can approach
we should take a different tone something like:

Prefabs allow higher levels of abstraction to be created facilitating customers to reuse functionality across their projects - both web, mobile. Prefabs in such cases include implementation of UI in accordance with form  factor. Partials allow such functionality to be implemented in a Prefab using different "partials" that include functionality targeted towards a particular form-factor.